.github/workflows/common.yml.inactive

```diff
@@ -28,6 +28,8 @@ on:
   # you guessed it - when releases are made.
   release:
     types: [published]
+  schedule:
+    - cron: '21 13 * * 1,3,5'
```

Comment:
Could we make it earilier so when India/Europeans folks come online they could be addressing this issue.

I think this is At 01:21 PM, only on Monday, Wednesday, and Friday according to https://crontab.cronhub.io/

Reply:
We can, this will need to be done each individual repo. No consequences of setting it here.

Reply:
okay lets set at individual repo - i didn't realize this could be so tailored.  very cool